sammba/registration/struct.py

```diff
@@ -777,15 +777,18 @@ def anats_to_template(anat_filenames, head_template_filename, write_dir,
 
     if dilated_head_mask_filename is None:
         out_clip_level = clip_level(in_file=head_template_filename)
-        out_threshold = threshold(in_file=head_template_filename,
-                                  thresh=out_clip_level.outputs.clip_val)
-        out_mask_tool = mask_tool(in_file=out_threshold.outputs.out_file,
+        out_calc_threshold = calc(
+            in_file_a=head_template_filename,
+            in_file_b=head_template_filename,
```

Comment:
Can't in_file_a be used on its own? Or is in_file_b a required flag to calc(), hence the need to input the same image twice?

Reply:
no it can, sorry I will correct